cpu/atmega_common/cpu.c

```diff
@@ -110,28 +120,30 @@ void cpu_init(void)
  * EIFR – External Interrupt Flag Register
  * PCIFR – Pin Change Interrupt Flag Register
  */
-ISR(BADISR_vect){
-
+ISR(BADISR_vect)
+{
     _reset_cause();
 
     printf_P(PSTR("FATAL ERROR: BADISR_vect called, unprocessed Interrupt.\n"
                   "STOP Execution.\n"));
 
     printf("IRQ_STATUS %#02x\nIRQ_STATUS1 %#02x\n",
-            (unsigned int)IRQ_STATUS, (unsigned int)IRQ_STATUS1 );
+            (unsigned int)IRQ_STATUS, (unsigned int)IRQ_STATUS1);
 
-    printf("SCIRQS %#02x\nBATMON %#02x\n", (unsigned int)SCIRQS, (unsigned int)BATMON );
+    printf("SCIRQS %#02x\nBATMON %#02x\n", (unsigned int)SCIRQS, (unsigned int)BATMON);
 
-    printf("EIFR %#02x\nPCIFR %#02x\n", (unsigned int)EIFR, (unsigned int)PCIFR );
+    printf("EIFR %#02x\nPCIFR %#02x\n", (unsigned int)EIFR, (unsigned int)PCIFR);
 
     /* White LED light is used to signal ERROR. */
     LED_PORT |= (LED2_MASK | LED1_MASK | LED0_MASK);
```

Comment:
Right now ```jiminy-mega256rfr2``` is the only board with this CPU, but the LED code should probably be guarded by board ifdefs and not CPU ifdefs to prevent future weirdness if another board with that CPU is added.

It would be helpful to have a common function to signal an error using an indicator (such as an LED), which is defined on a per-board basis. Potential idea for a future PR.

Reply:
Agree, I cannot only remove it though, since jiminy board relies on this to signal a failure. I think I'll need another PR to solve this before this one.